### ISSUE TITLE
Please package this as an MSIX installer.

### ISSUE THREAD

**Issue by 42837531** [abdelkader/vCardEditor#6]
That shall allow people who are forced to use Windows 10 S to use this on their devices without worrying that their method of installation is not supported. It is also a vastly more standardised and superior method of installation than EXE files, MSI files, and APPX files.

**Comment by mokhin-denis** [comment]
Is it about [this](https://learn.microsoft.com/ru-ru/windows/msix/overview?ranMID=43674&ranEAID=FE4O7wtxe6g&ranSiteID=FE4O7wtxe6g-FrE3Kti1GeLChFSSLLZW4Q&epi=FE4O7wtxe6g-FrE3Kti1GeLChFSSLLZW4Q&irgwc=1&OCID=AIDcmm549zy227_aff_7795_1243925&tduid=(ir__0zemgjnnzokfbm3xloumqekomv2xekn3xzxc1omn00)(7795)(1243925)(FE4O7wtxe6g-FrE3Kti1GeLChFSSLLZW4Q)()&irclickid=_0zemgjnnzokfbm3xloumqekomv2xekn3xzxc1omn00)? And is it simple to use [this](https://learn.microsoft.com/en-us/windows/msix/desktop/desktop-to-uwp-packaging-dot-net)?

**Comment by RokeJulianLockhart** [comment]
> https://github.com/abdelkader/vCardEditor/issues/6#issuecomment-1610717198

@mokhin-denis, indeed, that's what I'm referring to. I can't attest to how trivial it would be, because I'm unfamiliar with your application's architecture. However, I know of projects which managed to create a package in approximately a week.
